src/js/tabs/configuration.js

```diff
@@ -311,6 +311,26 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         // translate to user-selected language
         i18n.localizePage();
 
+        var gyro_align_content_e = $('.tab-configuration .gyro_align_content');
+        var legacy_gyro_alignment_e = $('.tab-configuration .legacy_gyro_alignment');
+        var legacy_accel_alignment_e = $('.tab-configuration .legacy_accel_alignment');
+
+        // Hide the new multi gyro element by default
+        gyro_align_content_e.hide();
+
+        // If we are sent USE_MULTI_GYRO flag from the target, show the new element, while hiding the old ones.
+        if (SENSOR_ALIGNMENT.use_multi_gyro == 1) {
```

Comment:
This is not in accordance with what Betaflight 4.0.0-RC6 is sending for `use_multi_gyro`.